src/generic_ui/scripts/user.ts

```diff
@@ -41,46 +40,61 @@ module UI {
       }
       this.name = profile.name;
       this.imageData = profile.imageData || UI.DEFAULT_USER_IMG;
-      this.isOnline = profile.isOnline;
     }
 
     // Returns two strings, where each matches an array name in model.contacts.
     // Does not use an enum because we need a string value, and typescript
     // enums evaluate to numbers.
```

Comment:
Although enums are numbers, they are 1:1 with strings. see: http://www.typescriptlang.org/Handbook#basic-types-enum. It's best to use enums when possible, and convert to strings only when needed. Why do we need strings here? If there's not a great reason, we should add a TODO and a cleanup issue.